packages/fields/__template/test/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Field test.
+ */
+
+import * as Blockly from 'blockly';
+import '../src/index.js';
+
+Blockly.defineBlocksWithJsonArray([
+  {
+    'type': 'test_field',
+    'message0': 'test: %1',
+    'args0': [
+      {
+        'type': 'field_template',
+        'name': 'FIELDNAME',
+        'value': 0, // TODO: change default value
+        'alt':
+        {
+          'type': 'field_label',
+          'text': 'NO_SLIDER_FIELD',
+        },
+      },
+    ],
+    'style': 'math_blocks',
+  }]);
+
+/**
+ * DOM loaded.
```

Comment:
I would like for us to follow the google style guide for wording method comments, although we are not consistently following this rule in Blockly Core. ("written in the third person, as if there is an implied This method ... before it")
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments
In this case, could reword to: Loads the DOM.